scripts/agent_app.py

```diff
@@ -26,102 +25,118 @@
 )
 from prediction_market_agent.agents.think_thoroughly_agent.deploy import (
     DeployableThinkThoroughlyAgent,
+    DeployableThinkThoroughlyProphetResearchAgent,
 )
 from prediction_market_agent.tools.streamlit_utils import (
     add_sink_to_logger,
     streamlit_escape,
 )
 
 AGENTS: list[
-    t.Type[DeployableKnownOutcomeAgent] | t.Type[DeployableThinkThoroughlyAgent]
-] = [DeployableKnownOutcomeAgent, DeployableThinkThoroughlyAgent]
+    type[DeployableKnownOutcomeAgent]
+    | type[DeployableThinkThoroughlyAgent]
+    | type[DeployableThinkThoroughlyProphetResearchAgent]
+] = [
+    DeployableKnownOutcomeAgent,
+    DeployableThinkThoroughlyAgent,
+    DeployableThinkThoroughlyProphetResearchAgent,
+]
 
 add_sink_to_logger()
 
-st.title("Agent's decision-making process")
 
-with st.sidebar:
-    streamlit_login()
+def agent_app() -> None:
```

Comment:
Multiprocessing errors out, if the streamlit application is not inside of a function and behind a `if __name__ == "__main__"` block.